NitroxPatcher/Patches/Dynamic/GoalManager_OnCompletedGoal_Patch.cs

```diff
@@ -10,17 +10,26 @@ public class GoalManager_OnCompletedGoal_Patch : NitroxPatch, IDynamicPatch
 {
     private static readonly MethodInfo TARGET_METHOD = Reflect.Method(() => GoalManager.main.OnCompleteGoal(default, default));
 
+    private static bool isNewGoal;
```

Comment:
The "send" data from prefix to postfix `__state` is better suited: https://harmony.pardeike.net/articles/patching-prefix.html#passing-state-between-prefix-and-postfix